.github/workflows/release.yml

```diff
@@ -39,11 +41,40 @@ jobs:
       - name: Build package
         run: python -m build
 
+      - name: Publish to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ contains(github.ref, '-rc') }}
+        with:
+          repository_url: https://test.pypi.org/legacy/
+          print_hash: true
+          skip_existing: true
+          verbose: true
+
+      # - name: Publish to PyPI
+      #   uses: pypa/gh-action-pypi-publish@release/v1.5
+      #   if: ${{ !contains(github.ref, '-rc') }}
+      #   with:
+      #     print_hash: true
+
+      - name: Parse version and output PyPI URL
+        id: version
+        # run littlepay first, so `littlepay -v` doesn't create the config file (which outputs a message)
+        run: |
+          pip install -e .
+          littlepay
+          output=$(littlepay -v)
+          version=${output/littlepay /}
+          version=${version/.0/.}
```

Comment:
Something to note: PyPI will actually redirect to the non-0-padded-month version if you try to go to the 0-padded one, e.g. https://pypi.org/project/eligibility-api/2023.09.1 works and goes to the page for `2023.9.1`.

So this line isn't entirely necessary, but still good to have I guess

Reply:
I'd be OK with removing. Your call.